synapse/storage/databases/main/appservice.py

```diff
@@ -172,15 +175,18 @@ async def set_appservice_state(self, service, state) -> None:
             "application_services_state", {"as_id": service.id}, {"state": state}
         )
 
-    async def create_appservice_txn(self, service, events):
+    async def create_appservice_txn(
+        self,
+        service: ApplicationService,
+        events: List[EventBase],
+        ephemeral: Optional[JsonDict] = None,
+    ) -> AppServiceTransaction:
         """Atomically creates a new transaction for this application service
         with the given list of events.
```

Comment:
Should this say anything about the ephemeral events coming in? It doesn't seem to get persisted, that might be worth mentioning?

Reply:
Have mentioned a thing.